components/elm/bld/namelist_files/use_cases/20thC_transient.xml

```diff
@@ -30,5 +30,10 @@
 <stream_year_last_popdens  phys="elm" use_cn=".true."   >2010</stream_year_last_popdens>
 <model_year_align_popdens  phys="elm" use_cn=".true."   >1850</model_year_align_popdens>
 
+<!-- Hack to override files for ne1024 for SCREAM, even though the sim years do not match up -->
+<fsurdat hgrid="ne1024np4.pg2">lnd/clm2/surfdata_map/surfdata_ne1024pg2_simyr2010_c211021.nc</fsurdat>
+<flanduse_timeseries hgrid="ne1024np4.pg2">lnd/clm2/surfdata_map/landuse.timeseries_ne1024pg2_historical_simyr1990-2014_c240109.nc</flanduse_timeseries>
+<check_finidat_year_consistency>.false.</check_finidat_year_consistency>
+<check_dynpft_consistency>.false.</check_dynpft_consistency>
```

Comment:
Would you want the two statements above to be false by default, even for non-SCREAM simulations? Can we add the `hgrid="ne1024np4.pg2"` condition to these lines?

Reply:
Yeah, good point. For what it's worth, these lines were already in the 20thC_CMIP6_transient.xml file.

Reply:
Thanks. I didn't know that there was a separate xml file for CMIP6, but figured someone had kept those consistency checks for some reason.

Reply:
The 'false' flags are just to allow more tolerance when there is a mismatch. finidat_year should be ok for the the planned AMIP as the initial year will match. dynpft_consistency needs to be false for the the SCREAM amip because we set to use an finidat that was spun-up  using simyr2010 settings.